src/geo/transform.js

```diff
@@ -380,7 +380,12 @@ class Transform {
      * @param {TileCoord} tileCoord
      * @param {number} maxZoom maximum source zoom to account for overscaling
      */
+
     calculatePosMatrix(tileCoord: TileCoord, maxZoom: number) {
+        return new Float32Array(this._calculatePosMatrix(tileCoord, maxZoom));
+    }
+
+    _calculatePosMatrix(tileCoord: TileCoord, maxZoom: number) {
```

Comment:
What's happening here? It seems like the no-underscore version essentially copies the matrix twice (note the ` new Float32Array` on line 399).